lms/djangoapps/ytp/management/commands/update_user_fullname.py

```diff
@@ -59,12 +59,19 @@ def handle(self, *args, **options):
                                    api_data = r.ok and r.json() or {}
                                    if api_data:
                                        full_name = (api_data.get('field_full_name', {}) or {}).get('und', [{}])[0].get('value', '')
+                                       first_name = (api_data.get('field_first_name', {}) or {}).get('und', [{}])[0].get('value', '')
+                                       last_name = (api_data.get('field_last_name', {}) or {}).get('und', [{}])[0].get('value', '')
+                                       if not full_name and (first_name or last_name ):
```

Comment:
lines L64 and L68 look opposite to each other. Please provide some general flow.